DontGetVorimed/DontGetVorimed.lua

```diff
@@ -0,0 +1,42 @@
+--[[
+    DontGetVorimed
+    Authors:
+    Ellomenop (Discord: ellomenop#2254)
+    Makes the first boon reward offer all 4 core boons
+]]
+ModUtil.RegisterMod("DontGetVorimed")
+
+local config = {
+    ModName = "Dont Get Vorimed",
+    Enabled = true
+}
+DontGetVorimed.config = config
+
+if ModConfigMenu then
+  ModConfigMenu.Register(config)
+end
+
+-- When the first room is created, set the number of loot choices to 4
+ModUtil.WrapBaseFunction("ChooseStartingRoom", function ( baseFunc, currentRun, roomSetName )
+  if config.Enabled then
+    LootChoiceExt.Choices = 4
```

Comment:
What if the first room offers a hammer?

Reply:
it doesn't matter, this just sets the config on first room as a convenient hook.  If we have a consistent "run is starting" hook I can change it.

Basically this config will persist until its explicitly reset